clients/web/src/views/body/PluginsView.js

```diff
@@ -63,16 +63,30 @@ var PluginsView = View.extend({
             this.failed = _.has(settings, 'failed') ? settings.failed : null;
             this.render();
         } else {
+            const promises = [
+                restRequest({
+                    path: 'system/plugins',
+                    type: 'GET'
+                }),
```

Comment:
I'd tack on a
```
.then( (resp) => resp )
```
to the end of both of these `restRequest` calls here. That will transform the both promises to have only a single argument (dropping the extra `textStatus`, `jqXHR` arguments).

Then below, inside the `.done((plugins, managePlugins) => {` block, you can refer to the arguments directly, without the need for a confusing `[0]` index.